apps/rule-manager/client/src/ts/utils/rule.ts

```diff
@@ -1,9 +1,17 @@
 import {DraftRule} from "../components/hooks/useRule";
 import {IconColor} from "@elastic/eui/src/components/icon";
 
-type RuleState = "live" | "archived" | "draft";
+export type RuleState = "live" | "archived" | "draft" | "error";
```

Comment:
Have added an extra type here on the client-side - "error" - to surface a buggy state more clearly. We could also choose to handle `RuleState | undefined`.